bucket/rainmeter.json

```diff
@@ -1,14 +1,23 @@
 {
     "homepage": "https://rainmeter.net/",
+    "description": "Rainmeter is a desktop customization tool for Windows.",
     "version": "4.3.0.3298",
     "license": "GPL-2.0-only",
+    "url": "https://github.com/rainmeter/rainmeter/releases/download/v4.3.0.3298/Rainmeter-4.3.exe",
+    "hash": "1bb57129f6871c7e5a835ded406de7096aebbe53196006ca0d00073948fbb16b",
     "architecture": {
         "64bit": {
-            "url": "https://github.com/rainmeter/rainmeter/releases/download/v4.3.0.3298/Rainmeter-4.3.exe#/Rainmeter.7z",
-            "hash": "1bb57129f6871c7e5a835ded406de7096aebbe53196006ca0d00073948fbb16b"
+            "installer": {
+                "script": "Expand-7zipArchive -Overwrite All \"$dir\\$fname\" \"$dir\""
```

Comment:
```suggestion
                "script": "Expand-7zipArchive -Overwrite All \"$dir\\$fname\" \"$dir\" -Removal"
```

Reply:
`$dir` parameter should not be needed.

https://github.com/lukesampson/scoop/blob/6141e46d6ae74b3ccf65e02a1c3fc92e1b4d3e7a/lib/decompress.ps1#L46